backend/cfg/cfg_equivalence.ml

```diff
@@ -307,6 +298,15 @@ let check_prim_call_operation :
          && Lambda.eq_mode expected_mode result_mode ->
     (* CR xclerc for xclerc: also check debug info *)
```

Comment:
Why is it a CR? Isn't it simply `Debuginfo.compare expected_dbginfo result_dbginfo` ?

Reply:
(This is dead code, which has probably not bee
run for quite some time - if anything, I would
rather delete this module.)